workflows/Discord_Release_Notify.yml

```
@@ -0,0 +1,31 @@
+name: Discord Release Notify
+
+on: 
+  release:
+    types:
+      - published
+
+jobs:
+  notify:
+    if: github.repository == 'AlternativeFFFF/Alt-F4'
+    name: Notify
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout PR
+      uses: actions/checkout@master
+    - name: Discord notification (1/2)
+      env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK_RELEASE }}
+        DISCORD_USERNAME: 'ALT-F4 Bot'
+        DISCORD_AVATAR: 'https://raw.githubusercontent.com/AlternativeFFFF/Alt-F4/master/assets/GLOBAL/img/spider.png'
+      uses: Ilshidur/action-discord@0.2.0
+      with:
+        args: "{{ EVENT_PAYLOAD.release.name }}"
+    - name: Discord notification (2/2)
+      env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
```

Probably forgot to update the webhook for the second message?
```suggestion
        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK_RELEASE }}
```